packages/remark/readme.md

```diff
@@ -8,125 +8,150 @@
 [![Backers][backers-badge]][collective]
 [![Chat][chat-badge]][chat]
 
-[**unified**][unified] processor to parse and serialize markdown.
-Built on [micromark][].
-Powered by [plugins][].
-Part of the [unified][] collective.
-
-*   API by [**unified**][unified]
-*   Parses markdown to a syntax tree with [`remark-parse`][parse]
-*   [**mdast**][mdast] syntax tree
-*   [Plugins][] transform the tree
-*   Serializes syntax trees to markdown with [`remark-stringify`][stringify]
-
-Don’t need the parser?
-Or compiler?
-[That’s OK: use **unified** directly][unified-usage].
+**[unified][]** processor with support for parsing markdown input and
+serializing markdown as output.
+
+## Contents
+
+*   [What is this?](#what-is-this)
+*   [When should I use this?](#when-should-i-use-this)
+*   [Install](#install)
+*   [Use](#use)
+*   [API](#api)
+    *   [`remark()`](#remark-1)
+*   [Syntax](#syntax)
+*   [Syntax tree](#syntax-tree)
+*   [Types](#types)
+*   [Compatibility](#compatibility)
+*   [Security](#security)
+*   [Contribute](#contribute)
+*   [Sponsor](#sponsor)
+*   [License](#license)
+
+## What is this?
+
+This package is a [unified][] processor with support for parsing markdown input
+and serializing markdown as output by using unified with
+[`remark-parse`][remark-parse] and [`remark-stringify`][remark-stringify].
+
+Please see the [monorepo readme][remark] for what the remark ecosystem is.
+
+**unified** is a project that transforms content with abstract syntax trees
+(ASTs).
+**remark** adds support for markdown to unified.
+**mdast** is the markdown AST that remark uses.
+
+## When should I use this?
+
+You can use this package when you want to use unified, have markdown as input,
+and want markdown as output.
+You can alternatively use unified itself and use [`remark-parse`][remark-parse]
+and [`remark-stringify`][remark-stringify] to do the same.
+You should also use unified directly with either of those plugins when the input
+you have or output you want is not markdown.
```

Comment:
Perhaps it could be a tiny bit clearer on when to use what and that remark is literally just unified under the hood. I think a common misconception is that `remark` is a different interface, not compatible with the unified interface. 

```suggestion
remark is unified configured specifically for markdown. Therefor you can alternatively use unified itself and use [`remark-parse`][remark-parse] and [`remark-stringify`][remark-stringify] to do the same.
if you want to bridge formats, such as [markdown to HTML][remark-rehype], it is recommended to do the above instead of using this package.
```

Reply:
Yeah wasn't clear to me at first either.  Always my problem with Unified/Remark, I go work on other things.  Come back and i'm always trying to unravel which interface is doing what.  And trying to remap my context around `unified <> remark <> remark-parse`.  Which it worked out I came into this _cold_ because it allowed me to look at the docs as a developer that hasn't looked at them in while.

For my use case, I'm purely just parsing markdown and storing the `mdast` ast trees for future processing pipelines (I keep the json shape).  It took me a few minutes to fully grok why my current project is using `unified/remark-parse` and not `remark` and it is because I don't want to `stringify`.  So not sure if that helps to give some context here in how it is explained to the larger developer audience?  But sometimes it can be helpful to explain something by what it doesn't do?

Reply:
The first sentence of the suggestion (`remark is unified configured specifically for markdown.`) feels like a repeat of the sentence before it?

How about:

```suggestion
It’s a shortcut for `unified().use(remarkParse).use(remarkStringify)`, which
you can also do manually.
When the input isn’t markdown (meaning you don’t need `remark-parse`) or the
output is not markdown (you don’t need `remark-stringify`), it’s also
recommended to use unified directly.
```

Reply:
Even more explicit, sounds good